### DISCUSSION TITLE
Switch user with focus mode

### DISCUSSION THREAD

**Discussion by miguelaht** [community/community#84080] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.142.1 (118013759)
Device: iPhone15,4 (iOS 17.2.1)
</details>

It'd be nice to be able to automatically switch between users based on the current focus mode (iOS).

**Comment by codesmith-emmy** [comment]
I would suggest you go to the repository and [Create an Issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). 🙂

**Comment by esahmed21** [comment]
Hi @miguelaht 👋 

Thank you for your suggestion! We're always looking for ways to enhance user experience. Could you please provide more details about how you envision the automatic switch between users based on focus mode working? This will help us better understand your needs and relay your feedback to the team effectively. Feel free to share any specific scenarios or preferences you have in mind. We appreciate your input!"

Thank you for getting in touch!
